### PR TITLE
Test title subtitle pagetitle xg

### PR DESCRIPTION
**Pull request by guanxiaotongyz**

### CODE REVIEW

File: src/app/demo/demo.component.spec.ts [hunk 1]
```diff
@@ -23,4 +26,341 @@ describe("DemoComponent", () => {
...
+  it("test editable defalut page title ", () => {
+     const { debugElement } = fixture;
+     const title1 = debugElement.query(By.css('#spanid')).nativeElement;
+     expect(title1.textContent).toContain('ReadAlong Studio');
+  });
+});
+
+// === test editable test page after edited ===
+describe("DemoComponent-pagetitle-edittvalue", () => {
+  let component: DemoComponent;
```
Comment (Iostream3100) [inline src/app/demo/demo.component.spec.ts:71]: Please put all the tests for this DemoComponent into one `describe` function for readability.

**Comment by Iostream3100** [comment]
Good job!
Please follow the recommendations of Conventional Commits https://www.conventionalcommits.org/ as much as possible in our projects, like "feat: implement a nav-bar"

**Comment by Iostream3100** [comment]
Please test the latest version(with input component) of the demo page, you can pull from the `editable_titles` branch.

**Review by Iostream3100 — CHANGES_REQUESTED** [review]
Please test the latest version(with input component) of the demo page, you can pull from the editable_titles branch.

**Comment by crystaljesica** [comment]
Please see those conflict files, you may need to git pull from the repo to get the latest file and make changes based on those.

**Pull request closed without merging**